### DISCUSSION TITLE
Is it possible to make spaceBetween parameter accept other units value than px?

### DISCUSSION THREAD

**Discussion by permanar** [nolimits4web/swiper#5179] [Ideas]
https://swiperjs.com/swiper-api#parameters

Hi guys.
So the problem rising when I want to add a space using a _vw_ unit instead of _px_.

I tried to use a hard-coded string value inside it but its not working (I later read the docs that it said only accepts number value).

Is it possible to achieve?

**Comment by nolimits4web** [comment]
In this case you need to use `slidesPerView: 'auto'` and use CSS `margin-right` on slides like here https://codesandbox.io/s/swiper-default-forked-4b215?file=/index.html

**Reply by permanar** [reply]
Hello @nolimits4web , 

Thank you for the fast reply and the kind solution given.

I actually have tried it but here's the issue that I faced (kinda hard to explain).
Here's a screencast video link: https://watch.screencastify.com/v/0PVBXbLCUTlitWxL3cC8

So basically, what I want is when on mobile (or probably this also affect the desktop one) is kinda having that a little peek of the 'next/previous card preview' thing. I used the _spaceBetween_ property and it looks pretty good (but I don't know is it the right way to do it).

But when I switch it to normal CSS margin, seems like the translate() a little bit broke. It has the same value as before (used _vw_ unit - _30px_ on _1920px_ viewport).

So yeah, basically all I want is that small little peek thing. 
Is there any workaround for my issue?

Thank you so much. I really appreciate the time given!
